### PR TITLE
[Merged by Bors] - chore: release 0.10.8

### PR DESCRIPTION
**Pull request by nacardin**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,58 @@
 # Release Notes
 
-## Platform Version 0.10.8 - UNRELEASED
+## Platform Version 0.10.9 - UNRELEASED
```
Comment (digikata) [inline CHANGELOG.md:3]: Did git cliff add this UNRELEASED header?  I think we should leave it out for the release and put it back in post release w/ the version bump.

**Review by digikata — APPROVED** [review]
LGTM other than the 0.10.9 header

File: CHANGELOG.md (hunk 1 above)
Comment (nacardin) [reply CHANGELOG.md]: yes, cliff did add it. just removed

**Review by sehz — APPROVED** [review]

**Pull request closed without merging**